tests/Http/Middleware/RateLimitTest.php

```diff
@@ -162,8 +162,8 @@ public function testRateLimitingWithLimitsSetOnRoute()
         $this->assertArrayHasKey('x-ratelimit-limit', $response->headers->all());
         $this->assertArrayHasKey('x-ratelimit-remaining', $response->headers->all());
         $this->assertArrayHasKey('x-ratelimit-reset', $response->headers->all());
-        $this->assertSame(4, $response->headers->get('x-ratelimit-remaining'));
-        $this->assertSame(5, $response->headers->get('x-ratelimit-limit'));
+        $this->assertSame(4, (int) $response->headers->get('x-ratelimit-remaining'));
```

Comment:
Hi! @mycarrysun, can you use this?
```php
        $this->assertSame('4', $response->headers->get('x-ratelimit-remaining'));
        $this->assertSame('5', $response->headers->get('x-ratelimit-limit'));
```

because possible of losing data using typecasting

Reply:
Yup, just pushed!